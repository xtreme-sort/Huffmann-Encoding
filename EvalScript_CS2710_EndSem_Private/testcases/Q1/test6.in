BuildTreeFromFrequencyTable
48
L 4
O 16
M 16
f 31
b 13
y 40
x 12
l 2
n 5
I 35
r 47
u 13
T 21
p 3
t 6
v 18
a 25
_ 36
H 25
J 30
C 34
0 45
S 44
G 13
Y 1
K 50
Q 4
V 22
7 31
W 29
c 11
d 42
6 37
g 29
e 43
h 26
j 10
P 46
k 34
q 3
9 39
E 46
F 32
R 4
A 43
D 14
3 24
i 3
Quit
