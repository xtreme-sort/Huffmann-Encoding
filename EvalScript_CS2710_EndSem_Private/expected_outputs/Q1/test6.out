(((((H,a),(((t,(i,p)),G),h)),(((b,u),(D,((q,(Y,l)),(L,Q)))),(W,g))),(((J,7),(f,F)),(((M,O),C),(k,I)))),((((_,6),((v,((R,n),j)),9)),((y,d),(A,e))),((((T,V),S),(0,E)),((P,r),(((c,x),3),K)))));
